### ISSUE TITLE
bginfo -- add package parameters

### ISSUE THREAD

**Issue by dtgm** [dtgm/chocolatey-packages#202]
Integrate @flcdrg's [rejected](https://chocolatey.org/packages/bginfo-startup/1.0.0) package into [bginfo](https://chocolatey.org/packages/bginfo)

Source files: https://github.com/dtgm/chocolatey-packages/tree/master/automatic/bginfo

**Comment by dtgm** [comment]
>I would love to generate the .bgi file from PowerShell, but as far as I know it can only be generated by running bginfo.exe interactively.

I think we shouldn't include a custom bgi file until we can figure a way to some way generate it dynamically with powershell. If we do, you want to at least change the path that points to `C:\User\dg...` within the file.

**Comment by flcdrg** [comment]
I suppose the other option would be to have a parameter that we can point to an external .bgi file (via URL) to download and use.

The custom.bgi file i had was a slightly tweaked version. If we did include a bgi it could just be a copy of the default settings created by bginfo.

**Comment by flcdrg** [comment]
My repo is here - https://github.com/flcdrg/chocolatey-bginfo-startup

**Comment by dtgm** [comment]
We probably are not able to include the bgi file anyway since it is a binary. MS is strict about this; from the SysInternals license:

> You may not [...] publish the software for others to copy;
https://technet.microsoft.com/en-us/sysinternals/bb469936

Of course a parameter to point to an external resource would work, but our own controls say chocolatey packages can't download software binary files from any other place than the official distribution location. It's meant to keep chocolatey as secure as we can. However, we could run the bginfo program from the install script to generate the binary. That's really our only option as I see it.

**Comment by dtgm** [comment]
@flcdrg Do you want me to add you as a maintainer to bginfo? It's an auto-package, so you would need to issue a PR back to this repo so your changes could be integrated in future auto-builds of versions released by SysInternals. (I check for updates on all my packages automatically several times a day.)

You would want to maintain the same schema of just changing the last number to the date for any package updates like:

4.21.0.20160210
4.21.0.20160526
etc

**Comment by flcdrg** [comment]
That would be great to be maintainer. No worries with the version schema.

I'm no lawyer, but I wouldn't have thought the restriction would apply to a bgi file. It's a data file that bginfo generates (similar to a docx with Word) - not something you download from sysinternals.com separately. But if you think we shouldn't include one then at all then we could just use one from a parameter if provided, and not use a bgi if none was set.

**Comment by dtgm** [comment]
1. You can now push packages against `bginfo`
2. You're probably correct that it is safe to distribute the bgi.

**Comment by flcdrg** [comment]
Slowly getting up to speed with Ketarin and the auto package stuff. One question - how do you go about overriding the version number as you suggested?

**Comment by tunisiano187** [comment]
@gep13 , you can close this issue as the package is handled by other maintainers

**Comment by flcdrg** [comment]
That would be me 😀. Yes, go ahead and close this. Maintained in https://github.com/flcdrg/au packages now.

**Comment by gep13** [comment]
@tunisiano187 thanks for noticing that this issue can be closed, and thanks to @flcdrg for helping with the maintenance of this package.

I have just added a commit here: https://github.com/dtgm/chocolatey-packages/commit/d424bef39a070ac4b0d465245770b33a68bc19fe to remove the packaging files and ketarin file for the bginfo package.

If we are closing issues, can we make sure that the necessary files have been moved from this repository, so that there aren't any issues with the automation in this project causing problems.  Thanks
